content/docs/beyond-the-basics/compliance-policies/index.md

```diff
@@ -4,11 +4,371 @@
+ 
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v1/compliance_policies
+```
+{{< /tab >}}}
+{{< /tabs >}}
+
+3. ***COMPLIANCE_PERIOD_OUTSTANDING:*** checks if the time between correlated events does not exceed set threshold.
+
+To correlate Events, define the attribute `arc_correlation_value` in the Event Attributes and set it to the same value on each pair of events that are to be associated.
+
+For example, "a Maintenance Request Event must be answered by a Maintenance Performed Event within 72 hours".
+
+{{< tabs name="compliance_period_outstanding" >}}
+{{{< tab name="UI" >}}
+{{< img src="CompliancePeriodAsset.png" alt="Rectangle" caption="<em>Add Asset Information</em>" class="border-0" >}}
+{{< img src="CompliancePeriodCompliance.png" alt="Rectangle" caption="<em>Add Compliance Information</em>" class="border-0" >}}
+
+{{< /tab >}}
+{{< tab name="YAML" >}}
+```yaml
+---
+steps:
+  - step:
+      action: COMPLIANCE_POLICIES_CREATE
+      description: Create COMPLIANCE_PERIOD_OUTSTANDING policy
+      print_response: true
+    description: There should not be outstanding Maintenance Requests for longer than 72hr
+    display_name: Outstanding Maintenance Requests 72hr
+    compliance_type: COMPLIANCE_PERIOD_OUTSTANDING
+    asset_filter:
+      - or: [ "attributes.arc_home_location_identity=locations/<location-id>" ]
+    event_display_type: Maintenance Requests
+    closing_event_display_type: Maintenance Performed
+    time_period_seconds: "259200"
+```
+Use the [archivist_runner](https://python.rkvst.com/runner/index.html) command to run your YAML file!
+ 
+```bash
+$ archivist_runner \
+      -u https://app.rkvst.io \
+      --client-id <your-client-id> \
+      --client-secret <your-client-secret> \
+      <path-to-yaml-file>
+```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+```json
+{
+    "compliance_type": "COMPLIANCE_PERIOD_OUTSTANDING",
+    "description": "There should be no outstanding Maintenance Requests longer than 72hr",
+    "display_name": "Outstanding Maintenance Requests 72hr",
+    "asset_filter": [
+        { "or": ["attributes.arc_home_location_identity=locations/<location-id>"]}
+    ],
+    "event_display_type": "Maintenance Requests",
+    "closing_event_display_type":  "Maintenance Performed",
+    "time_period_seconds": "259200"
+}
+```
+Use the curl command to run your JSON file! See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here.
+ 
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v1/compliance_policies
+```
+{{< /tab >}}}
+{{< /tabs >}}
+
+4. ***COMPLIANCE_DYNAMIC_TOLERANCE:*** checks if time between correlated events does not exceed defined variability.
```

Comment:
Might be cleaner to say “…is not excessively different to the observed average normal duration for similar events” or something like that. I worry that “defined variability” is a bit opaque

Reply:
Agreed, change made